app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
@chloealee in this instance is `$item` a collection? (in which case you need to do something like in step 4) You would probably have to do a check like::

```
Pseduo code:

if $item is an instance of Item {
 return $this->item($item, $code, $this->itemTransformer);
} else {
return $this->collection($item, $code, $this->itemTransformer);
}
```